src/client/components/input-field/index.css

```diff
@@ -1,3 +1,10 @@
 .input-field {
   display: inline-block;
 }
+
+.rwd-table .input-field table tr {
```

Comment:
Fixing for a specific component is a temporary solution (a workaround for a specific case) and should be avoided.